chain-information-rpc.json.example

```diff
@@ -0,0 +1,9 @@
+{
+  "chainName": "<CHAIN NAME>",
+  "chainId": ######,
+  "platformChainId": "<P CHAIN ID>",
```

Comment:
why is the `platformChainId` included here? This is a constant across all networks

Reply:
This is the 'subnet chain id on the P-Chain', as opposed to 'chain id on the EVM' (the line 3 field).
So, `q2aTwKuyzgs8pynF7UXBZCU7DejbZbZ6EUyHr3JQzYgwNPUPi` vs `53935` for example. Neither refers to the P-Chain itself.

Reply:
Can we call these as defined on the P-Chain the `subnetID` and `blockchainID` for consistency and then move VM specific information into its own field such as `vmSpecificInformation: {...}` which could then include the current `chainId` field.

I'd strongly prefer to avoid canonicalizing this as a misnomer.